rag/web_crawler/scraper.py

```
@@ -0,0 +1,106 @@
+# Uses the crawl4ai Python library to scrape website content into Markdown files
+import os
+import logging
+import asyncio
+from crawl4ai import AsyncWebCrawler # Assuming this is the correct import
+from crawl4ai import CrawlerRunConfig, AsyncWebCrawler, CacheMode
```

Ruff (0.12.2)

Redefinition of unused `AsyncWebCrawler` from line 5

Remove definition: `AsyncWebCrawler`

(F811)